RecoLocalTracker/SiPixelClusterizer/plugins/PixelThresholdClusterizer.cc

```diff
@@ -138,7 +146,7 @@ void PixelThresholdClusterizer::clusterizeDetUnitT( const T & input,
 	  
 	  //  Check if the cluster is above threshold  
 	  // (TO DO: one is signed, other unsigned, gcc warns...)
-	  if ( cluster.charge() >= theClusterThreshold) 
+	  if ( cluster.charge() >= clusterThreshold) 
```

Comment:
actually given that cluster.charge()  is a int please use a int for clusterThreshold!
moving to int the computation of cluster.charge()  was a major speed-up improvement